package.json

```diff
@@ -49,6 +49,7 @@
     "@types/jest-plugin-context": "^2.9.2",
     "@types/joi": "^14.3.3",
     "@types/lodash": "^4.14.144",
+    "@types/express-serve-static-core": "4.17.20",
```

Comment:
As per message earlier, can you put a bit more information as to why you have had to pull in this dependency please.